typeDefs.js

```diff
@@ -83,6 +93,41 @@ const typeDefs = gql`
     name: [Language]
     images: ImageSet
     available: Boolean
+    simpleName: String
+  }
```

Comment:
Please, go through all product's schema definition and setup non-nullable marks for all fields that cannot be null inside database

Reply:
done